a8c-files.php

```diff
@@ -86,13 +86,15 @@ public function __construct() {
 		// ensure the correct upload URL is used even after switch_to_blog is called
 		add_filter( 'option_upload_url_path', array( $this, 'upload_url_path' ) );
 
-		// Conditionally schedule the attachment filesize metaata update job
-		if ( defined( 'VIP_FILESYSTEM_SCHEDULE_FILESIZE_UPDATE' ) && true === VIP_FILESYSTEM_SCHEDULE_FILESIZE_UPDATE ) {
-			// add new cron schedule for filesize update
-			add_filter( 'cron_schedules', array( $this, 'filter_cron_schedules' ), 10, 1 ); // phpcs:ignore WordPress.WP.CronInterval.CronSchedulesInterval
-
-			// Schedule meta update job
-			$this->schedule_update_job();
+		if ( defined( 'WP_CLI' ) && WP_CLI ) {
+			// Conditionally schedule the attachment filesize metadata update job
+			if ( defined( 'VIP_FILESYSTEM_SCHEDULE_FILESIZE_UPDATE' ) && true === VIP_FILESYSTEM_SCHEDULE_FILESIZE_UPDATE ) {
+				// add new cron schedule for filesize update
+				add_filter( 'cron_schedules', array( $this, 'filter_cron_schedules' ), 10, 1 ); // phpcs:ignore WordPress.WP.CronInterval.CronSchedulesInterval
```

Comment:
Hmm, I think we want this to occur even outside of Cron though. Events are "purged" when they are invalid at various places in WP, or during registration need to use a valid schedule, etc. Best to leave it globally set I think

Reply:
0.o

```
// Not actually five minutes; we want it to run faster though to get through everything.
$schedule['vip_five_minutes'] = [ .... ]
```

Reply:
@WPprodigy the first comment was addressed, and the second one - yeah 0.o :)

Reply:
That's why you never put the interval in the `cron_schedule` name :)